### DISCUSSION TITLE
Utopia, Eutopia and Protopia: what term to use ...

### DISCUSSION THREAD

**Discussion by rufuspollock** [life-itself/community#522] [General]
### Traditionally, our kind of aspirations would be considered utopian

Our aspiration for a radically wiser, weller world, to an [awakening society] would be termed "utopian".

[awakening society]: https://lifeitself.org/awakening-society

### Utopia and utopianism seems to have gotten a bad name

There seems to be a sense that utopia is a difficult term to use: harmed by its association either with naivety or authoritarianism. On the naive side utopia is a thing for dreamers and hippies. It's unrealistic, unattainable, an impossible dream. On the authoritarian side: utopias were what the communists (or even Nazis) promised and their dreams became nightmares. In this case, utopias are dangerous temptations that lead us to engage in and justify things we shouldn't -- to make an omelette [a utopia] you've got to break a few eggs [kill a few people!]. In short utopias become dystopias.

### Can we heal the term utopia?

Personally, I've long been an advocate for resurrecting and healing the term "utopia". Perhaps, by combining it with a term like "pragmatic" to emphasize that this is a sober, realistic, even cautious, effort -- see https://lifeitself.org/blog/2017/10/20/pragmatic-utopians and https://lifeitself.org/blog/2020/12/21/pragmatic-utopianism.[^2]

[^2]: in fact, even before Life Itself formally existed I and @lkavanagh30 started a website called "Utopian Cast" and published   

> I like to describe myself as a pragmatic utopian – and Life Itself as an exercise in pragmatic utopianism. Utopianism seems to have a bad name today. I wrote this as a response to that presumption of guilt during the first Life Itself nucleus retreat.
> 
> We talk of smart cities but not wise ones. We are caught in a trap of small visions, and a disappearance of hope.
> 
> We are utopians. We should be able to say this proudly and without explanation but that is not yet so.

### Or should we adopt an alternative like the emerging term Protopia?

At the same time, perhaps utopia is simply too tarnished to be recoverable.

And there is a growing advocacy for the term "protopian". For example, see this long essay by Hanzi https://metamoderna.org/whats-the-difference-between-utopia-eutopia-and-protopia/ covering utopia, eutopia and protopia and advocating for protopia cf it's subtitle:

> Why Modern Utopia Fails — But Metamodern Protopia Might Not

Or the One Project who are using the term Protopia by default e.g. in the essay by their founder https://oneproject.org/architecture-of-abundance/

> Humanity has no roadmap from today to what futurist Kevin Kelly calls Protopia, a world that gets progressively better every year — more healthy, just, safe, creative, and fun. What we do know is that calls for structural change and the terminal flaws in the status quo are now impossible to ignore. Some will choose to double down on the thinking of the past. Some will seek to patch it up.

### What do you think?

I'd like to hear what others think!

- What term to you think most appealing out of utopian, eutopia, protopia?
- Do you think there is another good term to describe the kind of aspirations for a radically wiser, weller world?
- Any other thoughts or reflections?

**Comment by msantolini** [comment]
Very interesting question @rufuspollock! I cannot but think of alternate terms, as I am always a bit hesitant with the potential framing of a universal scale of progress. I was thinking of something like "entopia" (doesn't exist i guess), to talk about the place "within", rather than our typical idea of an "outer" place as the single topos of development. Something along the line of the alchemical wisdom "As Above So Below; As Within So Without". Otherwise i also thought of "syntopia" as an integral-flavored term, that would emphasize the integration of perspectival topi, or of the inner and outer.
Anyway, I don't know a lot about the Freinacht discussion so I might be missing out on existing discussions on this! But just adding some intuitive thoughts on this. 🙏

**Comment by Alex-Goodall** [comment]
Hanzi's article, as always, is well worth the time and effort to read.

I very much like his mapping of Utopia / Eutopia / Protopia onto Modern / Postmodern / Metamodern.

At the end of the article, he provides 6 distinctions between the three terms. One I particularly like is:

The **Utopian** envisions a linear path towards a brighter future. 
The **Eutopian** believes that pearls of happy and sane society are found precisely by looking away from the trodden path. 
(You make this point very nicely at the end of your _Pragmatic Utopianism_ article, Rufus - with which I resonate a lot: especially the Being parts.)
The **Protopian** envisions a non-linear pathway to a wider range of possibilities, a chaotic or [strange attractor](https://en.wikipedia.org/wiki/Attractor), which in itself constitutes the wider map of potential futures (or the “phase space” of society’s evolution), but where it is impossible to foresee which future will actually occur. So: from linear, to non-linear, to chaotic thinking.

**Reply by rufuspollock** [reply]
Really appreciate this summary @Alex-Goodall 🙏

**Reply by msantolini** [reply]
Indeed! Resonates a lot with the "collective adaptation" framing of the SFI crowd (https://osf.io/preprints/socarxiv/5f2ad/).

**Reply by Alex-Goodall** [reply]
Thanks for that link, Marc. Very interesting to see the nature of that research.
